02_bubble/srini/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var outWriter io.Writer = os.Stdout
+
+func bubble(sliceToSort []int) []int {
+	sliceLen := len(sliceToSort)
+
+	for i := 0; i < sliceLen-1; i++ {
+		stop := true
```

Comment:
Choose a name that is closer to the meaning. One possibility is `changed := false`.